aten/src/ATen/native/mkldnn/MKLDNNCommon.h

```diff
@@ -5,6 +5,11 @@
 
 #if AT_MKLDNN_ENABLED()
 #include <ideep.hpp>
+#ifndef IDEEP_PREREQ
+#define IDEEP_PREREQ(major, minor, patch, revision) \
+  (((IDEEP_VERSION_MAJOR << 32) + (IDEEP_VERSION_MINOR << 16) + (IDEEP_VERSION_PATCH << 8) + (DNNL_VERSION_PATCH)) \
```

Comment:
Can we add a comment here for the semantic of `IDEEP_VERSION_MAJOR `, `IDEEP_VERSION_MINOR`, `IDEEP_VERSION_PATCH` and `DNNL_VERSION_PATCH`? Should `IDEEP_VERSION_MAJOR `, `IDEEP_VERSION_MINOR` be the same as `OneDNN` major version and minor version?

Reply:
Thanks for the comment. The definitions of these macros are here: https://github.com/intel/ideep/blob/ideep_pytorch/include/ideep.hpp

Reply:
From above link, looks like `IDEEP_VERSION_PATCH` is same as `DNNL_VERSION_PATCH`?

Reply:
Yes. We check ideep API change by `IDEEP_VERSION_REVISION`. A comment is added and a typo is also fixed there.